front/app/utils/helperUtils.ts

```diff
@@ -105,6 +106,8 @@ export function isPage(pageKey: pageKeys, pathName: string) {
       return pathnameWithoutLocale.startsWith('/sign-in');
     case 'sign_up':
       return pathnameWithoutLocale.startsWith('/sign-up');
+    case 'pages_menu':
+      return pathnameWithoutLocale.includes('/pages-menu');
```

Comment:
Maybe making it a bit more explicit with `pathnameWithoutLocale.includes('/admin/pages-menu')` would be nice, but not necessary.

Reply:
Good point, I'll make the change